Training 453/app/images/logo.svg

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>

```

Comment:
if this asset is not used, please remove it. Try always to keep the project as clean as possible